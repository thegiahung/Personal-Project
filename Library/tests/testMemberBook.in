Member books 100000
Add member Python
Add collection books.csv
Member books 100000
Member books 100003
Rent 100000 111111
Member books 100000
Rent 100000 111112
Member books 100000
Relinquish 100000 111112
Member books 100000
Relinquish 100000 111111
Member books 100000
Exit
