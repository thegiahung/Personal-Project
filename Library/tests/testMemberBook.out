user: No members in system.

user: Success.

user: 17 books successfully added.

user: Member not currently renting.

user: No such member in system.

user: Success.

user: To Kill a Mockingbird (Harper Lee)

user: Success.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)

user: Success.

user: To Kill a Mockingbird (Harper Lee)

user: Success.

user: Member not currently renting.

user: Ending Library process.
